### ISSUE TITLE
Error "too few arguments to function" with SetLightColor, SetAmbientColor, and SetLightDir

### ISSUE THREAD

**Issue by Androxx00** [Yackerw/StarDust-DS#4]
Hi, I'm trying to get this to compile before I start creating what I want to make, but I keep getting the error "too few arguments to function" with SetLightColor, SetAmbientColor, and SetLightDir. I originally cloned the repo so I have the latest commits before I started working on things. Here's the log:

$ make
make -C arm7
make[1]: Entering directory '/opt/StarDust-DS/source/arm7'
main.c
C:/devkitPro/msys2/opt/StarDust-DS/source/arm7/source/main.c:10:9: warning: "SOUND_FREQ" redefined
   10 | #define SOUND_FREQ(n)   ((-0x1000000 / (n)))
      |         ^~~~~~~~~~
In file included from C:/devkitPro/libnds/include/nds.h:284,
                 from C:/devkitPro/msys2/opt/StarDust-DS/source/arm7/source/main.c:1:
C:/devkitPro/libnds/include/nds/arm7/audio.h:49:9: note: this is the location of the previous defini
tion
   49 | #define SOUND_FREQ(n)   (-soundTimerFromHz(n))
      |         ^~~~~~~~~~
linking StarDustDS.elf
make[1]: Leaving directory '/opt/StarDust-DS/source/arm7'
make -C arm9
make[1]: Entering directory '/opt/StarDust-DS/source/arm9'
main.c
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/main.c: In function 'main':
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/main.c:65:9: error: too few arguments to funct
ion 'SetLightColor'
   65 |         SetLightColor(RGB15(16, 16, 16));
      |         ^~~~~~~~~~~~~
In file included from C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/sdcollision.h:5,
                 from C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/main.c:7:
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/sdrender.h:252:6: note: declared here
  252 | void SetLightColor(int lightId, char R, char G, char B);
      |      ^~~~~~~~~~~~~
In file included from C:/devkitPro/libnds/include/nds/arm9/background.h:87,
                 from C:/devkitPro/libnds/include/nds.h:253,
                 from C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/main.c:2:
C:/devkitPro/libnds/include/nds/arm9/video.h:133:23: warning: unsigned conversion from 'int' to 'cha
r' changes value from '8456' to '8' [-Woverflow]
  133 | #define RGB15(r,g,b)  ((r)|((g)<<5)|((b)<<10))
      |                       ^~~~~~~~~~~~~~~~~~~~~~~~
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/main.c:66:25: note: in expansion of macro 'RGB
15'
   66 |         SetAmbientColor(RGB15(8, 8, 8));
      |                         ^~~~~
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/main.c:66:9: error: too few arguments to funct
ion 'SetAmbientColor'
   66 |         SetAmbientColor(RGB15(8, 8, 8));
      |         ^~~~~~~~~~~~~~~
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/sdrender.h:258:6: note: declared here
  258 | void SetAmbientColor(char R, char G, char B);
      |      ^~~~~~~~~~~~~~~
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/main.c:67:9: error: too few arguments to funct
ion 'SetLightDir'
   67 |         SetLightDir(0, -4096, 0);
      |         ^~~~~~~~~~~
C:/devkitPro/msys2/opt/StarDust-DS/source/arm9/source/sdrender.h:250:6: note: declared here
  250 | void SetLightDir(int lightId, f32 x, f32 y, f32 z);
      |      ^~~~~~~~~~~
make[2]: *** [/opt/devkitpro/devkitARM/base_rules:40: main.o] Error 1
make[1]: *** [Makefile:103: build] Error 2
make[1]: Leaving directory '/opt/StarDust-DS/source/arm9'
make: *** [Makefile:38: checkarm9] Error 2

**Comment by Yackerw** [comment]
Right, the repo isn't entirely up-to-date in its requirements. I'll fix this as soon as I finish the collision updates I'm working on.

**Comment by MJpixel03** [comment]
hi, same error
